hazelcast-sql/src/test/java/com/hazelcast/sql/impl/plan/cache/PlanCacheIntegrationTest.java

```diff
@@ -16,93 +16,90 @@
 
 package com.hazelcast.sql.impl.plan.cache;
 
-import com.hazelcast.client.test.TestHazelcastFactory;
 import com.hazelcast.config.IndexType;
 import com.hazelcast.core.HazelcastInstance;
 import com.hazelcast.map.IMap;
-import com.hazelcast.sql.SqlResult;
-import com.hazelcast.sql.SqlRow;
-import com.hazelcast.sql.impl.QueryException;
-import com.hazelcast.sql.impl.SqlResultImpl;
-import com.hazelcast.sql.impl.exec.FaultyExec;
-import com.hazelcast.sql.impl.exec.scan.MapScanExec;
-import com.hazelcast.sql.impl.plan.Plan;
-import com.hazelcast.test.HazelcastParallelClassRunner;
-import com.hazelcast.test.annotation.ParallelJVMTest;
+import com.hazelcast.sql.impl.optimizer.PlanKey;
+import com.hazelcast.sql.impl.optimizer.SqlPlan;
+import com.hazelcast.test.HazelcastSerialClassRunner;
 import com.hazelcast.test.annotation.QuickTest;
-import org.junit.After;
+import org.junit.Before;
+import org.junit.BeforeClass;
 import org.junit.Test;
 import org.junit.experimental.categories.Category;
 import org.junit.runner.RunWith;
 
+import java.util.concurrent.atomic.AtomicInteger;
+
+import static com.hazelcast.sql.impl.SqlTestSupport.nodeEngine;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNotSame;
-import static org.junit.Assert.assertNull;
 import static org.junit.Assert.assertSame;
-import static org.junit.Assert.fail;
 
-@RunWith(HazelcastParallelClassRunner.class)
-@Category({QuickTest.class, ParallelJVMTest.class})
+@RunWith(HazelcastSerialClassRunner.class)
+@Category(QuickTest.class)
 public class PlanCacheIntegrationTest extends PlanCacheTestSupport {
+    private final AtomicInteger mapNumber = new AtomicInteger(0);
```

Comment:
You can use `randomName()`.

Reply:
`randomName()` sometimes produces invalid symbols from SQL syntax perspective, and test fails.

Reply:
The one from `SqlTestSupport`?